src/main/java/com/internet/shop/dao/ProductDaoImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.internet.shop.dao;
```

Comment:
```suggestion
package com.internet.shop.dao.impl;
```
with the services the same